scripts/sct_deepseg_sc.py

```diff
@@ -176,8 +189,8 @@ def main():
         im_ctr.save(fname_ctr)
 
     if path_qc is not None:
-        generate_qc(fname_image, fname_seg=fname_seg, args=args, path_qc=os.path.abspath(path_qc),
-                    dataset=qc_dataset, subject=qc_subject, process='sct_deepseg_sc')
+        generate_qc(fname_image, fname_seg=fname_seg, args=sys.argv[1:], path_qc=os.path.abspath(path_qc),
+    dataset = qc_dataset, subject = qc_subject, process = 'sct_deepseg_sc')
```

Comment:
same comment about space

Reply:
Changes made in commit [d1ee3f9](https://github.com/neuropoly/spinalcordtoolbox/commit/d1ee3f9da6970a07d3a134ccec21240369785f19)